User-Guide.md

```diff
@@ -0,0 +1,171 @@
+# Introduction
+
+This Kata Containers User Guide aims to be a comprehensive guide to the explanation of,
+installation, configuration, and use of Kata Containers.
+
+The Kata Containers [source repositories](https://github.com/kata-containers) contain
+significant amounts of other documentation, covering some subjects in more detail.
+
+## What is Kata Containers?
+
+Kata Containers is, as defined in the [community repo](https://github.com/kata-containers/community):
+
+> Kata Containers is an open source project and community working to build a standard implementation of lightweight Virtual Machines (VMs) that feel and perform like containers, but provide the workload isolation and security advantages of VMs.
+
+It is a drop in additional OCI compatible container runtime, which can therefore be used
+with [Docker] and [Kubernetes].
+
+# Installation
+
+## Supported platforms
+
+Kata Containers is primarily a Linux based application. It can be installed on the most
+common Linux distributions, using the common Linux packaging tools.
+
+
+Details on installation can be found in [documentation repository](https://github.com/kata-containers/documentation/tree/master/install).
+
+For the curious, adventurous, developers or those using a distribution not presently
+supported with pre-built pacakges, Kata Containers can be
+[installed from source](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md). If you are on a distribution that is not presently supported, please feel free
+to reach out to the [community](https://github.com/kata-containers/community#community) to
+discuss adding support. Of course, contributions are most welcome.
```

Comment:
Link to https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md?